addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -380,17 +364,12 @@ public void packetReceived(byte[] packet) {
                 } else if (message instanceof RFXComTransmitterMessage) {
                     RFXComTransmitterMessage resp = (RFXComTransmitterMessage) message;
 
-                    byte seqNbr = getSeqNumber();
-                    if (resp.seqNbr == seqNbr) {
-                        logger.debug("Transmitter response received: {}", message.toString());
-                        setResponseMessage(resp);
-                        synchronized (notifierObject) {
-                            notifierObject.notify();
-                        }
-                    } else {
-                        logger.warn("Sequence number '{}' does not match, expecting number '{}'", resp.seqNbr, seqNbr);
-                    }
+                    logger.debug("Transmitter response received: {}", resp.toString());
 
+                    if (resp.response != Response.NAK)
+                        transmitQueue.sendNext();
+                    else
+                        transmitQueue.resend();
```

Comment:
Will you retry for ever? What if we are just sending a corrupt message to RFXCOM because of a bug or userinput? I think you will be locked for ever in that case?